cms/templates/js/license-selector.underscore

```diff
@@ -0,0 +1,106 @@
+                                        .replace("{license_name}", license.name)
+                            %>
+                        </a>
+                    <% } else { %>
+                        &nbsp;
+                    <% } %>
+                </p>
+            </li>
+        <% }) %>
+    </ul>
+
+<% var license = licenseInfo[model.type]; %>
+<% if(license && !_.isEmpty(license.options)) { %>
+    <div class="wrapper-license-options">
+        <label class="label setting-label" for="list-license-options">
+            <%- gettext("Options for {license_name}").replace("{license_name}", license.name) %>
+        </label>
+        <p class='tip tip-inline'>
+            <%- gettext("The following options are available for the {license_name} license.")
+                        .replace("{license_name}", license.name) %>
+        </p>
+        <ul class="license-options" id="list-license-options">
+            <% _.each(license.option_order, function(optionKey) { %>
+                <% var optionInfo = license.options[optionKey]; %>
+                <% if (optionInfo.type == "boolean") { %>
+                    <% var optionSelected = model.options[optionKey]; %>
+                    <% var optionDisabled = optionInfo.disabled ||
+                            (optionInfo.conflictsWith && _.any(optionInfo.conflictsWith, function(key) {return model.options[key];}));
+                    %>
+                    <li data-option="<%- optionKey %>"
+                            class="action-item license-button
+                                   <% if (optionSelected) { print("is-selected"); } %>
+                                   <% if (optionDisabled) { print("is-disabled"); } else { print("is-clickable"); } %>"
+                    >
+                        <i class="fa fa-fw
+                             <% if(optionSelected) { print("fa-check-square-o"); } else { print("fa-square-o"); } %>
+                             <% if(optionDisabled) { print("is-disabled"); } %>
+                             "></i>
+                        <h4 class="option-name"><%- optionInfo.name %></h4>
+                        <div class="explanation"><%- optionInfo.help %></div>
+                    </li>
+                <% } // could implement other types here %>
+            <% }) %>
+        </ul>
+    </div>
+<% } %>
+
+<% if (showPreview) { %>
+    <div class="license-preview-wrapper">
+        <label class="label setting-label" for="license-preview">
+            <%= gettext("License Display") %>
+        </label>
+        <p class="tip">
+            <%= gettext("The following message will be displayed at the bottom of the courseware pages within your course.") %>
+        </p>
+        <div id="license-preview">
+        <% // keep this synchronized with the contents of common/templates/license.html %>
+        <% if (model.type === "all-rights-reserved") { %>
+            © <span class="license-text"><%= gettext("All Rights Reserved") %></span>
```

Comment:
What is the right way to include a copyright symbol? Does it need to be translated? What happens with right-to-left? At the very least it should use ``&copy;`` I would think.